app/javascript/components/cloud-network-form/cloud-network-form.schema.js

```diff
@@ -0,0 +1,213 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+import { API } from '../../http_api';
+
+export const providersNetworkTypes = [
```

Comment:
I don't suppose there is a way to option this from API?

Reply:
There could be, but you still need to change this form according to the selected one of these values, so you still need to exactly know what values you can get and how to react. (If there were a new option, we would still need to go here and update the schema.)